crates/sui/tests/cli_tests.rs

```diff
@@ -326,6 +326,31 @@ async fn test_ptb_publish_and_complex_arg_resolution() -> Result<(), anyhow::Err
     Ok(())
 }
 
+#[sim_test]
+async fn test_ptb_publish() -> Result<(), anyhow::Error> {
+    move_package::package_hooks::register_package_hooks(Box::new(SuiPackageHooks));
+    let mut test_cluster = TestClusterBuilder::new().build().await;
+    let context = &mut test_cluster.wallet;
+    let mut package_path = PathBuf::from(TEST_DATA_DIR);
+    package_path.push("ptb_complex_args_test_functions");
+
+    let publish_ptb_string = format!(
+        r#"
+         --move-call sui::tx_context::sender
+         --assign sender
```

Comment:
I had never thought to do it like this before (I always passed in the active address as its own pure input!)

Reply:
Heh, just using @manolisliolios repro, so credit to him!